frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -141,6 +143,7 @@ watch(
                         v-bind="field"
                         class="w-100 custom-height"
                         @input="cleanupForestClientNumberInput()"
+                        @keypress.enter="verifyForestClientNumber(forestClientNumbersInput)"
```

Comment:
I don't see we need this. It looks like with malformed input you still allow frontend to hit the backend.

Reply:
Hello, Ian! @OlgaLiber2 suggested that pressing 'enter' to validate would be desirable

Reply:
Hi Jota, Both '**click the button**' and '**pressing enter**' are all fine. That's usability consideration.

However, you missed a point, it is about **wrong format of data hitting the backend**.
If we rely all on the backend checks, without frontend validations, then that's fine.

But our frontend can already check if the input entered at least follow certain format (length 8 and all digits) and validates it before sending to the backend.

Initially, without looking at the code, what I see is this:
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/e1990e1c-262e-40c0-a99d-efbab94d3de3)
So it means the code still has a bug.

Please see if you can find it.